lnwallet/channel.go

```diff
@@ -744,8 +763,8 @@ func (lc *LightningChannel) evaluateHTLCView(view *htlcView, ourBalance,
 			continue
 		}
 
-		processAddEntry(entry, ourBalance, theirBalance, nextHeight,
-			remoteChain, false)
+		processAddEntry(entry, ourBalance, theirBalance, satoshisSent,
```

Comment:
The calculation of satoshis sent/received should only be triggered when we encounter a non-add entry (right before we execute `processRemoveEntry`. Rght before we call `processRemoveEntry`, the sent/received stats can be updated based on if we're processing a local or remote view, and if the log were iterating through is the local log, or the remote log. 

HTLC adds shouldn't contribute to the sent/received stats as they only transfer satoshis into a "limbo" state which may be reverted (without settling) due to the cancellation of an HTLC for w/e reason.